dataactcore/utils/duns.py

```diff
@@ -270,32 +269,58 @@ def update_duns(sess, duns_data, table_name='duns', metrics=None, deletes=False,
             'updated_duns': []
         }
 
+    if key_cols is None:
+        key_cols = ['awardee_or_recipient_uniqu']
+    elif key_cols not in (['awardee_or_recipient_uniqu'], ['awardee_or_recipient_uniqu', 'uei']):
```

Comment:
If we're requiring DUNS and making UEI optional, why not always have DUNS and just have a boolean flag to include UEI? It seems simpler and cleaner overall.

Reply:
I think that makes sense. I'll name it something like `includes_uei`.